nodeup/pkg/model/context.go

```diff
@@ -344,19 +355,23 @@ func (c *NodeupModelContext) KubectlPath() string {
 }
 
 // BuildCertificatePairTask creates the tasks to pull down the certificate and private key
-func (c *NodeupModelContext) BuildCertificatePairTask(ctx *fi.ModelBuilderContext, key, path, filename string) error {
-	certificateName := filepath.Join(path, filename+".pem")
-	keyName := filepath.Join(path, filename+"-key.pem")
+func (c *NodeupModelContext) BuildCertificatePairTask(ctx *fi.ModelBuilderContext, name, dir, filename string) error {
+	certificateName := filepath.Join(dir, filename+".crt")
```

Comment:
This one was my fault :-) .. a habit of `.pem` and `-key.pem` .. its good to keep consistent .. the only concern would be for a roll out as it's used in a number of places 

```shell
  1 nodeup/pkg/model/secrets.go|48 col 14| if err := b.BuildCertificateTask(c, fi.CertificateId_CA, "ca.crt"); err != nil {                                                                                                                    
  2 nodeup/pkg/model/secrets.go|73 col 15| if err := b.BuildCertificateTask(c, name, "server.cert"); err != nil {
  3 nodeup/pkg/model/secrets.go|83 col 15| if err := b.BuildCertificateTask(c, "apiserver-proxy-client", "proxy-client.cert"); err != nil {
  4 nodeup/pkg/model/secrets.go|92 col 15| if err := b.BuildCertificateTask(c, "apiserver-aggregator", "apiserver-aggregator.cert"); err != nil {
  5 nodeup/pkg/model/secrets.go|101 col 15| if err := b.BuildCertificateTask(c, "apiserver-aggregator-ca", "apiserver-aggregator-ca.cert"); err != nil {
  6 nodeup/pkg/model/kube_apiserver.go|97 col 15| if err := b.BuildCertificateTask(c, name, name+".pem"); err != nil {
  7 nodeup/pkg/model/etcd_tls.go|42 col 15| if err := b.BuildCertificateTask(c, name, certificate); err != nil {
  8 nodeup/pkg/model/etcd_tls.go|48 col 15| if err := b.BuildCertificateTask(c, fi.CertificateId_CA, ca); err != nil {
  9 nodeup/pkg/model/context.go|351 col 14| if err := c.BuildCertificateTask(ctx, key, certificateName); err != nil {
 10 nodeup/pkg/model/context.go|358 col 4| // BuildCertificateTask is responsible for build a certificate request task
```
Doesn't look like it will effect anything .. as those should be altered on the nodeup .. Just have to be careful no daemosets or deployments deployed by manifests (`channels`) is referencing them.